crates/re_data_store/src/store_db.rs

```diff
@@ -237,6 +238,36 @@ impl StoreDb {
         }
     }
 
+    /// Helper function to create a recording from a [`StoreInfo`] and a some [`DataRow`]s.
+    ///
+    /// This is useful to programmatically create recordings from within the viewer, which cannot
+    /// use the `re_sdk`, which is not Wasm-compatible.
+    pub fn from_rows(
+        app_id: impl Into<ApplicationId>,
+        rows: impl IntoIterator<Item = DataRow>,
+    ) -> anyhow::Result<Self> {
+        let store_info = StoreInfo {
+            application_id: app_id.into(),
+            store_id: StoreId::random(StoreKind::Recording),
+            is_official_example: true,
+            started: Time::now(),
+            store_source: StoreSource::Viewer,
+            store_kind: StoreKind::Recording,
```

Comment:
These are all way too opinionated to be used in a function called just `from_rows`.

It is way too easy for someone to just call `StoreDb::from_rows(…)` and assume that will work great for blueprints, or non-official examples, or stuff that was recorded a long time ago, etc etc. Better to pass `StoreInfo` by argument.

Reply:
OK, will do. I went back and forth on this one and reached the opposite conclusion, but the less opinionated option prevails in case of doubt of course.

Side note for the future: it wouldn't be so "annoying" to expose `StoreInfo` if there wasn't so much data denormalisation with StoreId and StoreKind...